client/app/models/project-applicant.js

```diff
@@ -0,0 +1,12 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
```

Comment:
Ah, so in CRM, pas applicant entities are different from project applicant entities! CRM is weird.